app/components/autocomplete/index.js

```diff
@@ -2,15 +2,29 @@
 // See License.txt for license information.
 
 import {connect} from 'react-redux';
+import {Platform} from 'react-native';
+import DeviceInfo from 'react-native-device-info';
 
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
 
+import {getDimensions} from 'app/selectors/device';
+
 import Autocomplete from './autocomplete';
 
 function mapStateToProps(state) {
     return {
+        listHeight: listHeight(state),
         theme: getTheme(state)
     };
 }
 
+function listHeight(state) {
+    const {deviceHeight} = getDimensions(state);
+    let offset = Platform.select({ios: 130, android: 120});
+    if (DeviceInfo.getModel() === 'iPhone X') {
+        offset = 150;
+    }
+    return deviceHeight - offset;
+}
+
```

Comment:
I'm wondering if I should move all of this to `app/components/autocomplete/autocomplete.js` rather than having `listHeight` here and then redefining it in `app/components/autocomplete/autocomplete.js`. Thoughts?